### ISSUE TITLE
Accessibility Issue in Service Account Page

### ISSUE THREAD

**Issue by nirajCITZ** [bcgov/api-services-portal#797]
1)
Description:  Ensures the contrast between foreground and background colors meets WCAG 2 AAA enhanced contrast ratio thresholds
Helpurl:      https://dequeuniversity.com/rules/axe/4.7/color-contrast-enhanced?application=axeAPI
[
    {
        "any": [
            {
                "id": "color-contrast-enhanced",
                "data": {
                    "fgColor": "#1a5a96",
                    "bgColor": "#f1f1f1",
                    "contrastRatio": 6.3,
                    "fontSize": "10.5pt (14px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "7:1"
                },
                "relatedNodes": [
                    {
                        "html": "<body class=\"chakra-ui-light\">",
                        "target": [
                            "body"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 6.3 (foreground color: #1a5a96, background color: #f1f1f1, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 7:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<a class=\"chakra-breadcrumb__link css-5yp7oh\" href=\"/manager/namespaces\">Namespaces (newplatform)</a>",
        "target": [
            ".css-5yp7oh"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 6.3 (foreground color: #1a5a96, background color: #f1f1f1, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 7:1"
    }
]
![image.png](https://images.zenhubusercontent.com/61a121c4a7a3264dfdc530ef/9353bd41-060b-4757-89c0-599706f6de5f)

-----------------------------------------------------------------
2) Description:  Ensures the contrast between foreground and background colors meets WCAG 2 AAA enhanced contrast ratio thresholds
Helpurl:      https://dequeuniversity.com/rules/axe/4.7/color-contrast-enhanced?application=axeAPI
[
    {
        "any": [
            {
                "id": "color-contrast-enhanced",
                "data": {
                    "fgColor": "#1a5a96",
                    "bgColor": "#f1f1f1",
                    "contrastRatio": 6.3,
                    "fontSize": "10.5pt (14px)",
                    "fontWeight": "normal",
                    "messageKey": null,
                    "expectedContrastRatio": "7:1"
                },
                "relatedNodes": [
                    {
                        "html": "<body class=\"chakra-ui-light\">",
                        "target": [
                            "body"
                        ]
                    }
                ],
                "impact": "serious",
                "message": "Element has insufficient color contrast of 6.3 (foreground color: #1a5a96, background color: #f1f1f1, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 7:1"
            }
        ],
        "all": [],
        "none": [],
        "impact": "serious",
        "html": "<a class=\"chakra-breadcrumb__link css-5yp7oh\" href=\"/manager/namespaces\">Namespaces (newplatform)</a>",
        "target": [
            ".css-5yp7oh"
        ],
        "failureSummary": "Fix any of the following:\n  Element has insufficient color contrast of 6.3 (foreground color: #1a5a96, background color: #f1f1f1, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 7:1"
    }
]

![image.png](https://images.zenhubusercontent.com/61a121c4a7a3264dfdc530ef/a606e623-4698-4442-ad10-e03eaadda35f)
